docs/HACKING.md

```diff
@@ -36,6 +38,7 @@ To build and push the operator image, execute:
 export OPERATOR_IMAGE=quay.io/<insert username>/wmco:$VERSION_TAG
 hack/olm.sh build
 ```
+If there is a build issue, make sure you populate submodules with `git submodule update --init` command and retry again.
```

Comment:
Consider referencing the existing section [## Updating Git submodules](https://github.com/openshift/windows-machine-config-operator/pull/683/commits/fe3c117ac0069fa1e5c8f37f3fb7c6ebc8359c92#diff-b842e6ab4a95a695d9449d106f091e6a134d9eac8d2aee1cd8b169fcb6b3a98bR198)

Reply:
+1

Reply:
That section is confusing and uses updating in two different ways 1) updating as in adding new commits to point the submodules at a new HEAD, 2) updating as in updating your local submodules to their expected HEAD.

I'll take the blame for that :frowning_face: